subsys/suit/platform/sink_selector/src/suit_sdfw_sink_selector.c

```diff
@@ -162,6 +165,13 @@ int suit_sink_select(suit_component_t dst_handle, struct stream_sink *sink)
 			}
 
 			LOG_ERR("SDFW sink not enabled");
+		} else if (number == 2) {
```

Comment:
I think the magic numbers 1 and 2 should be defined as macros somewhere (`suit_metadata.h` seems to be a good place)